lib/hawkular/inventory/inventory_api.rb

```diff
@@ -25,9 +26,9 @@ def initialize(entrypoint = nil, credentials = {}, options = {})
     #   entrypoint: http://localhost:8080/hawkular/inventory
     # and another sub-hash containing the hash with username[String], password[String], token(optional)
     def self.create(hash)
-      hash[:entrypoint] ||= 'http://localhost:8080/hawkular/inventory'
       hash[:credentials] ||= {}
       hash[:options] ||= {}
+      fail 'no parameter ":entrypoint" given' if hash[:entrypoint].nil?
```

Comment:
this can move 2 lines up, because if it fails, no point to do the credentials/options initialization